py_swf/clients/workflow.py

```diff
@@ -65,3 +70,341 @@ def terminate_workflow(self, workflow_id, reason):
             workflowId=workflow_id,
             reason=reason,
         )
+
+    def count_open_workflow_by_time(self, oldest_start_date, latest_start_date=None):
+        """
+        start_date should be type of datetime
+        :param oldest_start_date: datetime
+        :param latest_start_date: datetime
+        :return:
+        """
+        return self._count_open_workflow_executions(
+            oldest_start_date=oldest_start_date,
+            latest_start_date=latest_start_date,
+        )
+
+    def count_open_workflow_by_type(self, name, oldest_start_date=None, latest_start_date=None, version=None):
```

Comment:
You are right, since they are optional params a clear docs string should suffice.